modules/clock/config/clock_config.h

```diff
@@ -31,5 +31,11 @@
 #define CLOCK_PER2_FREQ  CLOCK_CPU_FREQ
 #define CLOCK_PER4_FREQ  CLOCK_CPU_FREQ
 
+/** @brief Activates automatic run-time calibration of internal clock
```

Comment:
`@brief` should be separated from detailed information by starting a new paragraphe (empty line).
Also, details should be properly capitalized and punctuated, and only separated by one space from the comment block's `*`.
Other occurrences below.